cfgov/v1/tests/models/test_related_posts.py

```diff
@@ -148,6 +149,25 @@ def test_related_posts_blog_limit(self):
         self.assertNotIn('Newsroom', related_posts)
         self.assertNotIn('Events', related_posts)
 
+    def test_related_posts_and_filtering(self):
+        """
+        Tests whether related posts are retrieved if and only if they match ALL
+        of the tags on the calling page.
+        """
+
+        self.block.value['relate_posts'] = True
+        self.block.value['relate_newsroom'] = True
+        self.block.value['relate_events'] = True
```

Comment:
Ah, got it. Yeah I generally prefer the conditions required to understand a test to be in the test itself, but I understand the desire to reuse what's in `setUp`. Looks good, thanks for the explanation!